ostd/src/mm/vm_space.rs

```diff
@@ -323,7 +323,7 @@ impl CursorMut<'_, '_> {
     /// Map a frame into the current slot.
     ///
     /// This method will bring the cursor to the next slot after the modification.
-    pub fn map(&mut self, frame: Frame, prop: PageProperty) {
+    pub fn map(&mut self, frame: DynUFrame, prop: PageProperty) {
```

Comment:
After seeing `DynUFrame` and `DynUSegment` show up in `kernel/` for over 40 times and their central roles in `ostd` (like this API), I think we should make their names even shorter... How about simply `UFrame` and `USegment`?  We can simply remove their cousins, `DynFrame` and `DynSegment`, which do not appear to be quite useful.